### ISSUE TITLE
Version available vs version currently installed swapped fir Firefox Browser and Firefox Focus in ffupdater 78.1.1

### ISSUE THREAD

**Issue by bilemke** [Tobi823/ffupdater#453]
**Describe the bug**
ffupdater displays currently installed version as the Available version and the actual available version as  the Installed version for "Firefox Browser" and "Firefox Focus". It also attempts to download the version currently installed instead if the new version that is actually available. Possibly for other apps as well (I only checked apps I normally have added/installed).

**To Reproduce**
Add browser tlas a new application.  Check fir updates. See "Installed" vs "Available" and compare against actually installed version on device.

**Expected behavior**
Installed displays installed version, Available displays available version, update downloads available version.

**Error message from FFUpdater**
None, erroneous display/functionality only.

**Comment by Tobi823** [comment]
Thanks for the info, I will look into it (maybe in 1/2 days)

**Comment by Tobi823** [comment]
Bad news - I can't reproduce this error and I can't see any obvious bugs in the source code.

Maybe Mozilla published temporary a wrong version number or there is a hidden bug in the app.

My plan: I'm going to refactor the source code for the main view to make bugs more visible.

**Comment by bilemke** [comment]
Just noticed you updated/closed the issue. FFupdater wasn't showing signs of this issue for a while. And today I got to see it happen, and I think I know what might be happening behind the scenes.

Looks like it is competing updates between apps. Today I noticed it for Firefox Focus. Installed app is 117.0 final (non beta). FFUpdater is wanting to pull a 116.3.0 version. Looks like Goigle Play Syore updated it to 117.0 yesterday on the 22nd. FFUpdater is checking against Github (by the look of i) and finding the latest non-beta release is 116.3.0 from 7 days ago. Seems Play Store has a new version before Github release.
